### ISSUE TITLE
Page titles should include the date and period

### ISSUE THREAD

**Issue by mattab** [piwik/piwik#9556]
Goal of this issue is to improve the Page title `<title>` in Piwik reports, so that the page title gives to user the full context of the current page.

Page title is visible in the browser bar: 
![screenshot from 2016-01-19 12 06 42](https://cloud.githubusercontent.com/assets/466765/12404984/211c2142-bea5-11e5-804d-98ee8ef730a8.png)

In particular we would like to add the date and period:

* the title [of the All Websites report](http://demo.piwik.org/index.php?module=MultiSites&action=index&idSite=3&period=day&date=yesterday) should be `All Websites dashboard - Sunday, January 17th, 2009 - Web analytics reports - Piwik` (instead of `All Websites dashboard - Web analytics reports - Piwik`)
* the title of [any standard analytics report such as this one](http://demo.piwik.org/index.php?module=CoreHome&action=index&date=yesterday&period=week&idSite=7#?module=Live&action=indexVisitorLog&date=yesterday&period=week&idSite=7) should be `Piwik Forums - week January 11 – 17, 2016 - Web Analytics Reports - Piwik` (instead of `Piwik Forums - Web Analytics Reports - Piwik`)
